src/utils/telemetryUtils.ts

```diff
@@ -10,6 +10,12 @@ export namespace TelemetryUtils {
             if (arg.subscription.subscriptionId) {
                 properties[TelemetryProperties.subscriptionId] = arg.subscription.subscriptionId;
             }
+            // data plane has no subscriptionId but subscriptionPath.
+            else if (arg.subscription.subscriptionPath && arg.subscription.tenantId && arg.subscription.userId) {
+                properties[TelemetryProperties.dataPlaneRuntimeUrl] = arg.subscription.subscriptionPath;
+                properties[TelemetryProperties.dataPlaneTenantId] = arg.subscription.tenantId;
+                properties[TelemetryProperties.dataPlaneClientId] = arg.subscription.userId;
+            }
             if (arg.label) {
```

Comment:
For dataplane, will it also have `resourceName`?

Reply:
Actually `resourceName` is upper layer of Api Center Service, but when user add Api Center Data plane Api, only add the runtimeUrl from Api Center service, we can not fetch the `resourceName`

Reply:
yes, it will log the label too